### PR TITLE
PLAT-140826: Fix ui-test fail on PopupTabLayout

### PR DESCRIPTION
**Pull request by dongsuwon**
Enact-DCO-1.0-Signed-off-by: Dongsu Won (dongsu.won@lgepartner.com)

### Checklist
* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
- Need to fix ui-test fail on PopupTabLayout for changed ux.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
- Fix ui-test fail on PopupTabLayout for changed ux.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]:# (Related issues, references)
PLAT-140826

### CODE REVIEW

**Review by MikyungKim — CHANGES_REQUESTED** [review]
I'm still seeing the below failures. Please check.
```
 1) 5-way interaction should show the collapsed tabs when focus enters the content
[chrome 80.0.3987.87 Linux #0-14] expected [Function: isCollapsed] to equal true
[chrome 80.0.3987.87 Linux #0-14] AssertionError: expected [Function: isCollapsed] to equal true
[chrome 80.0.3987.87 Linux #0-14]     at Context.<anonymous> (/home/mikyung27.kim/github/sandstone/tests/ui/specs/PopupTabLayout/PopupTabLayout-specs.js:58:24)
[chrome 80.0.3987.87 Linux #0-14]
[chrome 80.0.3987.87 Linux #0-14] 2) 5-way interaction should expand the tabs when focus returns to the tabs
[chrome 80.0.3987.87 Linux #0-14] expected [Function: isCollapsed] to equal false
[chrome 80.0.3987.87 Linux #0-14] AssertionError: expected [Function: isCollapsed] to equal false
[chrome 80.0.3987.87 Linux #0-14]     at Context.<anonymous> (/home/mikyung27.kim/github/sandstone/tests/ui/specs/PopupTabLayout/PopupTabLayout-specs.js:73:24)
```

File: tests/ui/specs/PopupTabLayout/PopupTabLayout-specs.js [hunk 1]
```diff
@@ -40,6 +40,7 @@ describe('PopupTabLayout', function () {
 				it('should show the collapsed tabs when focus enters the content', function () {
```
Comment (MikyungKim) [inline tests/ui/specs/PopupTabLayout/PopupTabLayout-specs.js:40]:
```suggestion
				it('should show the collapsed tabs when a user enters a menu', function () {
```

File: tests/ui/specs/PopupTabLayout/PopupTabLayout-specs.js [hunk 2]
```diff
@@ -216,6 +210,21 @@ describe('PopupTabLayout', function () {
 
 				expect(actual).to.equal(expected);
 			});
+
+			it('should collapse tab only when user enters a menu ', function () {
+				Page.spotlightRight();
+				Page.delay(500);
+				expect(popupTabLayout.isCollapsed).to.be.false();
+
+				Page.spotlightRight();
```
Comment (MikyungKim) [inline tests/ui/specs/PopupTabLayout/PopupTabLayout-specs.js:219]: I think this step is not needed.

Comment (dongsuwon) [reply tests/ui/specs/PopupTabLayout/PopupTabLayout-specs.js:219]: As Jenkins performance problems, it needs to some delay.
If remove delay , It occur failure in TV test.

Comment (dongsuwon) [reply tests/ui/specs/PopupTabLayout/PopupTabLayout-specs.js]: Suggested change.

File: tests/ui/specs/PopupTabLayout/PopupTabLayout-specs.js (hunk 1 above)
Comment (dongsuwon) [reply tests/ui/specs/PopupTabLayout/PopupTabLayout-specs.js]: Suggested change.

**Review by MikyungKim — APPROVED** [review]
jenkins build # 1485 succeeded.
💯

**Pull request merged**